sklearn/metrics/tests/test_common.py

```diff
@@ -619,9 +619,9 @@ def test_invariance_string_vs_numbers_labels():
 
 
 def test_inf_nan_input():
-    invalids =[([0, 1], [np.inf, np.inf]),
-               ([0, 1], [np.nan, np.nan]),
-               ([0, 1], [np.nan, np.inf])]
+    invalids = [([0, 1], [np.inf, np.inf]),
+                ([0, 1], [np.nan, np.nan]),
+                ([0, 1], [np.nan, np.inf])]
 
     METRICS = dict()
     METRICS.update(THRESHOLDED_METRICS)
```

Comment:
This was also a result of the auto-pep8 in my atom text editor that corrected the spacing automatically. I have returned it as it was.